### PR TITLE
[zstd] update to 1.3.1

### PR DESCRIPTION
**Pull request by codicodi**

### CODE REVIEW

File: ports/zstd/portfile.cmake [hunk 1]
```diff
@@ -14,6 +14,11 @@ else()
     set(ZSTD_SHARED 1)
 endif()
 
+# Enable multithreaded mode. CMake build doesn't have dedicated option for this,
+# but it is the default in Makefile and VS projects.
+set(VCPKG_C_FLAGS "${VCPKG_C_FLAGS} /DZSTD_MULTITHREAD")
+set(VCPKG_CXX_FLAGS "${VCPKG_CXX_FLAGS}")
```
Comment (KindDragon) [inline ports/zstd/portfile.cmake]: CMake support it too https://github.com/facebook/zstd/commit/bf8a30ce0d0e8f2b45a954fcf396220050f9172d

Comment (codicodi) [inline ports/zstd/portfile.cmake]: You're right, I adjusted the comment

Comment (KindDragon) [inline ports/zstd/portfile.cmake]: Sorry, I think I miss something.
Why not just enable CMake option `ZSTD_MULTITHREAD_SUPPORT`?

https://github.com/facebook/zstd/blob/aecf3b479c45affa9fd8ead068e9160253a8ec5c/build/cmake/programs/CMakeLists.txt#L60-L61
https://github.com/facebook/zstd/blob/aecf3b479c45affa9fd8ead068e9160253a8ec5c/build/cmake/CMakeLists.txt#L31

Comment (codicodi) [inline ports/zstd/portfile.cmake]: `SET_PROPERTY(TARGET zstd...`
That only affects zstd executable, there is no way to build library with multithreading enabled.

**Comment by alexkaratarakis** [comment]
Thanks!

**Pull request merged**